corehq/apps/sms/migrations/0049_test_api_backend.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sms', '0048_delete_sqlicdsbackend'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
I think this should be a no-op migration unless `settings.UNIT_TESTING` is true. In other words, I don't think we actually want to create this model in production. I forget if a no-op migration is the best option to achieve that, and I'm not remembering any other pattern for test-specific migrations at the moment.

Reply:
Offlined with Daniel and instead of making a test model I am just using a production model that supports an API.